java-api-core/src/com/kinvey/java/sync/dto/SyncRequest.java

```diff
@@ -73,8 +75,16 @@ public static HttpVerb fromString(String verb){
     @Key("url")
     private String url;
 
+    @Key("requestMethod")
+    private String requestMethod;
 
-    public SyncRequest(){};
+    public SyncRequest(){}
+
+    public SyncRequest(RequestMethod requestMethod, SyncMetaData entityID, String collectionName) {
+        this.requestMethod = requestMethod.name();
+        this.id = entityID;
```

Comment:
What is the reason for putting entityId, customerVersion and customRequestHeaders in the same object? Is this how it was before?

Reply:
Yes, it was before, I've saved it.